packages/xo-web/src/xo-app/settings/remotes/index.js

```diff
@@ -74,6 +75,17 @@ const COLUMN_STATE = {
   ),
   name: _('remoteState'),
 }
+const COLUMN_DISK = {
+  itemRenderer: (remote, { formatMessage }) => (
+    <span>
+      {remote.info &&
```

Comment:
1. always use explicit test if not already a boolean
2. it does not work here because `Handler#getInfo()` always an object
3. what if the object does not contain `used` and `available`

Reply:
Still not resolved.

Reply:
What about this?

Reply:
It's ok now i think. At first, I was just testing `remote.info`, I added some explicit test.